cms/static/cms/sass/components/_pagetree.scss

```diff
@@ -4,24 +4,370 @@
+        top: 3px;
+        outline: none;
+        color: $white !important;
+        padding: 8px 10px !important;
+        margin-left: 5px;
+        border: none !important;
+        background: $color-primary !important;
+        &:hover {
+            background: darken($color-primary, 5%) !important;
+        }
+        &:active,
+        &:focus {
+            color: $white !important;
+            background: darken($color-primary, 15%) !important;
+        }
+    }
+    .cms-pagetree-header-filter {
+        position: relative;
+        float: left;
+        input {
+            width: $pagetree-header-search-width !important;
+            padding-right: 35px !important;
+            margin-top: 2px;
+            transition: width $speed-base;
+        }
+    }
+    // active state on focus or when opening the filters
+    .cms-pagetree-header-filter-active input {
+        @media (min-width: $pagetree-mobile) {
+            width: $pagetree-header-search-width * 2 !important;
+        }
+    }
+}
+.cms-pagetree-header-filter-trigger {
+    position: absolute;
+    right: 0;
+    top: 0;
+    cursor: pointer;
+    padding: 10px;
+    transform: rotate(270deg);
+    &:hover .cms-icon,
+    &:active .cms-icon,
+    &:focus .cms-icon {
+        color: $color-primary;
+    }
+    .cms-icon {
+        font-size: 11px;
+        color: $gray;
+    }
+}
+.cms-pagetree-header-filter-container {
+    display: none;
+    overflow: auto;
+    position: absolute;
+    left: 0;
+    top: 38px;
+    width: 100%;
+    padding: 5px 20px 15px;
+    max-height: $pagetree-header-filter-height;
+    box-shadow: $pagetree-box-shadow;
+    box-sizing: border-box;
+    border-radius: $border-radius-base;
+    background: $white;
+    h3 {
+        color: $gray;
+        font-size: 14px;
+        font-weight: normal;
+        text-transform: uppercase;
+        padding: 10px 0 0 0;
+    }
+    ul,
+    li {
+        list-style-type: none;
+        padding: 2px 0;
+        margin: 0;
+    }
+    // color states
+    li a {
+        color: $gray-light;
+    }
+    .selected a {
+        color: $color-primary;
+        font-weight: bold;
+    }
+}
+.cms-pagetree-header-search-close {
+    position: absolute;
+    right: 0;
+    top: 0;
+    padding: 10px;
+    &:hover .cms-icon,
+    &:active .cms-icon,
+    &:focus .cms-icon {
+        color: $color-primary;
+    }
+    .cms-icon {
+        font-size: 12px;
+        color: $gray;
+    }
+}
+.cms-pagetree-header-search-reset {
+    float: left;
+    font-size: 90%;
+    padding: 7px $pagetree-header-space;
+}
+
+// DROPDOWN
+.cms-pagetree-dropdown-menu {
+    float: right;
+    padding-left: 10px;
+    margin-left: $pagetree-header-space;
+    @media (min-width: $pagetree-mobile) {
+        float: left;
+    }
+
+    .cms-pagetree-dropdown-container {
+        right: -1px;
+        top: 30px;
+        min-width: $dropdown-width + 25px;
+        a {
+            padding: 5px $submenu-item-padding-horizontal;
```

Comment:
should probably not depend on the submenu variable. you can create pagetree-related variable that defaults to the submenu item related variable for example